test/job/MomoJobBuilder.spec.ts

```diff
@@ -1,49 +1,80 @@
 import { MomoJobBuilder } from '../../src/job/MomoJobBuilder';
 
 describe('MomoJobBuilder', () => {
-  it('can build a job with all attributes', () => {
+  it('can build a job with all attributes and an interval', () => {
     const momoJob = new MomoJobBuilder()
       .withName('name')
-      .withInterval('one minute')
-      .withFirstRunAfter(0)
+      .withSchedule('one minute', 0)
       .withConcurrency(1)
       .withMaxRunning(1)
       .withHandler(jest.fn())
       .build();
 
     expect(momoJob.name).toEqual('name');
-    expect(momoJob.interval).toEqual('one minute');
-    expect(momoJob.firstRunAfter).toEqual(0);
+    expect(momoJob.schedule).toEqual({ interval: 'one minute', firstRunAfter: 0 });
     expect(momoJob.concurrency).toEqual(1);
     expect(momoJob.maxRunning).toEqual(1);
     expect(momoJob.handler.toString()).toEqual(jest.fn().toString());
   });
 
-  it('can build only with required attributes', () => {
-    const momoJob = new MomoJobBuilder().withName('name').withInterval('one minute').withHandler(jest.fn()).build();
+  it('can build a job with all attributes and a cron schedule', () => {
```

Comment:
`can build a cron job with all attributes`  maybe?

analogous for the interval test above